collectors/meta_template/schema.json

```diff
@@ -0,0 +1,387 @@
+      "properties": {
+        "text": {
+          "type": "string"
+        }
+      }
+    },
+    "alerts": {
+      "type": "array",
+      "description": "This section is autogenerated, only user validation is to ensure the \"os\" field is in line with the OS the collector can run on. If this runs on freebsd but the \"os\" field is \"linux\", just delete the list item.",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "link": {
+            "type": "string"
+          },
+          "metric": {
+            "type": "string"
+          },
+          "info": {
+            "type": "string"
+          },
+          "os": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "metrics": {
+      "type": "object",
+      "description": "Autogenerated section,  user input is needed to provide a description for labels if any, and to add the scope's rendered name in each scope.",
+      "properties": {
+        "text": {
+          "type": "string"
+        },
+        "folding": {
+          "type": "object",
+          "description": "Controls whether or not to fold this section, in case the table is very large and we don't need to present it",
+          "properties": {
+            "title": {
+              "type": "string",
+              "description": "Title for the collapsed section"
+            },
+            "enabled": {
+              "type": "boolean",
+              "description": "Boolean value, controls whether to collapse or not"
+            }
+          }
+        },
+        "scopes": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string"
+              },
+              "rendered_name": {
+                "type": "string",
+                "description": "Please fill this in, it should fit:\n\"Metrics per X\" and \"X Level\" where X is the scope name."
+              },
+              "text": {
+                "type": "string",
+                "description": "Description for the scope."
+              },
+              "labels": {
+                "type": "array",
+                "items": {
+                  "name": {
+                    "type": "string"
+                  },
+                  "text": {
+                    "type": "string"
+                }
+              },
+              "metrics": {
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "name": {
+                      "type": "string"
+                    },
+                    "text": {
+                      "type": "string"
+                    },
+                    "unit": {
+                      "type": "string"
```

Comment:
Might be good to point out here (in a `description`) that the format of units is supposed to follow UCUM. We can also do this a bit later, but as soon as this lands, and the UI changes support them, I'll make sure that all metadata files will have the properly formatted units.

Reply:
I see this as a task that will take a lot more time for the first iteration, you would have to read through every single metric for every single module and fix the units..

Depends on how many resources we get for this to do it.

cc @sashwathn